.atomist/tests/project/TypeScriptEditorTest.feature

```diff
@@ -5,8 +5,8 @@ Feature: Make sure the sample TypeScript Editor has some tests
 
 
   Scenario: TypeScriptEditor should edit a project correctly
-    Given an empty project
+    Given a project with a certain file
     When the TypeScriptEditor is run
     Then parameters were valid
     Then changes were made
-    Then the hello file says hello
+    Then that certain file looks different
```

Comment:
Do you think adding a `Then` that uses a well-known step with parameters would be helpful? For example, `Then file at hello.txt exists`.